app/assets/stylesheets/components/common/_social_sharing.scss

```diff
@@ -6,52 +6,68 @@
 }
 
 .social-sharing__item {
-  height: 33px;
-  width: 55px;
   margin: 0 10px 0 0;
   border-radius: 3px;
   position: relative;
+  overflow: visible;
 
   &:last-child {
     margin-right: 0;
   }
 }
 
+.social-sharing__item:before {
+  visibility: hidden;
+}
+
+.social-sharing__item__icon {
+  width: 55px;
+}
+
 .social-sharing__item--facebook {
   background: $color-facebook;
 
-  &:hover {
+  & > .social-sharing__item__icon:hover,
+  & > .social-sharing__item__icon:focus {
     background: $color-facebook-hover;
   }
 
   .svg-icon--facebook,
   .icon--facebook {
-    margin: 5.5px auto;
+    height: 33px;
+    margin: 0 auto;
+    padding: 5.5px 0;
```

Comment:
Same for the padding: 5.5px 0. Can this be set to 6px?

Although half pixels are possible (in theory) on double density and 4K screens it's still best to let the browser render from a whole number.